canvas_modules/harness/cypress/support/canvas/node-cmds.js

```diff
@@ -14,9 +14,16 @@
  * limitations under the License.
  */
 
-Cypress.Commands.add("getNodeForLabel", (nodeLabel) =>
-	cy.get(getNodeGrpSelector())
-		.then((grpArray) => findGrpForLabel(grpArray, nodeLabel)));
+Cypress.Commands.add("getNodeForLabel", (nodeLabel) => {
```

Comment:
It's only trivial but could you change getNodeForLabel to be getNodeWithLabel  so it uses the same terminology as getCommentWithText ?